Testing/Unit/sitkElastixFilterTests.cxx

```diff
@@ -0,0 +1,309 @@
+#include "elxElastixFilter.h"
+#include "elxTransformixFilter.h"
+#include "elxParameterObject.h"
```

Comment:
Hrmm... looks like making the Elasitx header includes PRIVATE is working, as this include is not found when building the test.

Reply:
Indeed... making PRIVATE is working a little bit too good  🤔

Reply:
I don't really understand why, because I did add the ElastixTransformixWrappers include directory to SimpleITK_INCLUDE_DIRS: https://github.com/SimpleITK/SimpleITK/blob/3a3bfe8cfeae04d339a1dfa21fd96a283f894682/CMakeLists.txt#L140-L144

Reply:
Hmmm... I think these elastix tests should not be part of SimpleITK! Only the TransformixImageFilter and ElastixImageFilter from the `itk::simple` namespace should be tested here, not the ones from `elx`. Good catch, @blowekamp ! I'll make another force-push